algorithm/marine/socaincr2mom6.yaml.j2

```diff
@@ -2,27 +2,27 @@ geometry:
   mom6_input_nml: mom_input.nml
   fields metadata: ./fields_metadata.yaml
 
-date: '{{marine_window_begin}}'
+date: '{{ MARINE_WINDOW_BEGIN_ISO }}'
 
 layers variable: [hocn]
 
 increment variables: [tocn, socn, uocn, vocn, ssh]
 
 vertical geometry:
-  date: '{{marine_window_begin}}'
+  date: '{{ MARINE_WINDOW_BEGIN_ISO }}'
   basename: ./INPUT/
   ocn_filename: MOM.res.nc
   read_from_file: 1
 
 soca increment:
-  date: '{{marine_window_begin}}'
+  date: '{{ MARINE_WINDOW_BEGIN_ISO }}'
```

Comment:
will defer to JCB's father @danholdaway on style guidance but I think most of these JCB algorithm templates would look more like `{{ marine_window_begin_iso }}`

Reply:
Why are you YELLING at us G?

Reply:
# **I'M NOT YELLING.**

Reply:
> # **I'M NOT YELLING.**

Doesn't mixing cases add unnecessary confusion and back and forth?

You have to prescribe a mapping between G-W and JCB in GDASapp anyway so I don't think it's all the complicated to have singular case on the jcb side.

Reply:
lower-cased stuff in my last commit.